contracts/priv/notebooksd.yml

```diff
@@ -147,6 +147,181 @@ paths:
           $ref: '#/components/responses/ServerError'
         '500':
           $ref: '#/components/responses/ServerError'
+  /notebooks/share:
+    get:
+      summary: get all Share records in org
+      operationId: getAllShares
+      tags:
+        - Share
+      parameters:
+        - name: orgID
+          in: query
+          schema:
+            type: string
+          required: true
+        - name: notebookID
+          in: query
+          schema:
+            type: string
+          required: false
+      responses:
+        '200':
+          description: returns all Share records in org
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Shares'
+        '401':
+          $ref: '#/components/responses/ServerError'
```

Comment:
Shouldn't be ServerError. It should be Unauthorized.

Reply:
I thought so too but it looks like George went through and changed all of them to be "ServerError" in this commit: https://github.com/influxdata/openapi/commit/02f9592832d641eeb00d7ca7bc243693bebcf91d

Check this out: https://github.com/influxdata/openapi/blob/master/src/svc/notebooksd/paths/notebooks_id.yml#L20

Reply:
Confirmed with Greg that they all should be "ServerError" now. All the error types have the same openapi spec, so he consolidated all the schemas into one.

Reply:
😞  Resolving it for now.